distributed/protocol/pickle.py

```diff
@@ -23,36 +25,43 @@ def _always_use_pickle_for(x):
         return False
 
 
-def dumps(x):
+def dumps(x, *, buffer_callback=None):
     """ Manage between cloudpickle and pickle
 
     1.  Try pickle
     2.  If it is short then check if it contains __main__
     3.  If it is long, then first check type, then check __main__
     """
+    pickle_dumps = functools.partial(pickle.dumps, protocol=HIGHEST_PROTOCOL)
+    cloudpickle_dumps = functools.partial(cloudpickle.dumps, protocol=HIGHEST_PROTOCOL)
+    if pickle.HIGHEST_PROTOCOL >= 5:
```

Comment:
Nitpick for readability - should use either `HIGHEST_PROTOCOL` or `pickle.HIGHEST_PROTOCOL`, not both. Slight preference to keep namespaced version, but really doesn't matter.

Reply:
There's also a slight readability overhead IMO by building up nesting `functools.partial` calls. Slight preference to reorg to avoid this if possible, or handle by building up `kwargs` dicts instead.

Reply:
Thanks Jim! 😄

Yeah that slipped. Sorry about that. Will fix.

Good point. Will update.

Reply:
Fixed in recent commits. Also added a couple tests 🙂